rust_dev_preview/firehose/README.md

```diff
@@ -0,0 +1,58 @@
+# AWS SDK for Rust code examples for Kinesis Data Firehose
```

Comment:
Recommend trying out WRITEME, especially for new READMEs!

Reply:
ACK - this came from an OSS contributor, I'll replace it when I redo all the rust ones :)